tests/tamr_client/fake.py

```diff
@@ -101,3 +101,13 @@ def transforms() -> tc.Transformations:
         ],
         unified_scope=["//Comment\nSELECT *;"],
     )
+
+
+def attribute() -> tc.Attribute:
+    return tc.Attribute(
+        url=tc.URL(path="datasets/1/attributes/RowNum"),
+        name="RowNum",
+        type=tc.attribute.type.Array(tc.attribute.type.STRING),
```

Comment:
nit: I think `Array<String>` is the default type for attributes, so you can omit.

Even if you don't omit, we should have a `DEFAULT` alias for it somewhere that you can use. Though maybe `DEFAULT` is not a good enough name 🤷

Reply:
It looks like `type` is not an optional argument to `tc.Attribute` if that is what you mean by omitting.  I can pass `type=tc.attribute.type.DEFAULT` and that works.

Do we want `type` to be optional in instantiating a `tc.Attribute` and default to `DEFAULT`?

Reply:
Not sure. Let's just use `DEFAULT` explicitly here. We can decide on making `type` optional in follow-on work.